unitary/alpha/quantum_world.py

```diff
@@ -72,11 +72,21 @@ def add_object(self, obj: QuantumObject):
         """
         if obj.name in self.used_object_keys:
             raise ValueError("QuantumObject {obj.name} already added to world.")
-        self.used_object_keys.add(obj.name)
+        self.used_object_keys[obj.name]=obj
```

Comment:
Questionable formatting

Reply:
yapf-ed the file, but it picked up some additional formatting changes.

Reply:
Apparently we were using black before, e.g. https://github.com/quantumlib/unitary/pull/45